Feature/sr 13 add icons

TurchynykOleksii opened the pull request.

TurchynykOleksii committed:
SR-13: add className props & fix bad names

TurchynykOleksii committed:
Merge branch 'main' into feature/SR-13-add-icons

TurchynykOleksii committed:
SR-13: delete icon with App

Vlad-Moskalenko commented on src/theme/icons.const.tsx:
```diff
@@ -1,15 +1,18 @@
 
 const DEFAULT_IMG_PROPS: IconProps = {
+  classNames:'icon',
   color: '#000',
```
впиши сюди значення currentColor

TurchynykOleksii committed:
Merge branch 'main' into feature/SR-13-add-icons

TurchynykOleksii committed:
SR-13: refactor icons and images

1) Reworked the Import/Export icons
2) Distributed icons and pictures in folders

TurchynykOleksii committed:
SR-13: replaced the use of icons

TurchynykOleksii committed:
SR-13: fix path with svg

TurchynykOleksii committed:
Merge branch 'main' into feature/SR-13-add-icons

TurchynykOleksii committed:
SR-13: add Readme.MD and create img names

1) added the Readme.MD description of the icons
2) added a picture of the visual representation of icons
